src/main/java/javax/jmdns/impl/JmDNSImpl.java

```diff
@@ -467,7 +478,7 @@ private void openMulticastSocket(HostInfo hostInfo) throws IOException {
         if (_socket != null) {
             this.closeMulticastSocket();
         }
-        _socket = new MulticastSocket(DNSConstants.MDNS_PORT);
+        _socket = new MulticastSocket(getMulticastBindAddress(hostInfo));
```

Comment:
Before, the MulticastSocket received `DNSConstants.MDNS_PORT` as a parameter (for non-Windows machines), now it is `new InetSocketAddress(DNSConstants.MDNS_PORT)` - is this intended and required?

Reply:
There is no difference actually. If we open `MulticastSocket` constructor for `DNSConstants.MDNS_PORT` we could see this code:

```java
public MulticastSocket(int port) throws IOException {
    this(new InetSocketAddress(port));
}
```